src/checkstyle/checkstyle.xml

```diff
@@ -91,6 +91,7 @@
                              org.hamcrest.CoreMatchers.*,
                              org.hamcrest.Matchers.*,
                              org.hamcrest.MatcherAssert.*,
+                             org.hamcrest.core.Is.*,
```

Comment:
Any reason we're adding this package instead of just using `Matchers.is()` or `Matchers.isA()`?

Reply:
good catch!!! I'll change that.